### PR TITLE
Pass a flag about the first test page to client side

### PR DESCRIPTION
**Pull request by miherlosev**
This PR added information to the client side about the first tested page and necessary to performing resize actions (`canUseDefaultWindowActions`). It's a part of the https://github.com/DevExpress/testcafe/issues/3691 task.

### CODE REVIEW

File: src/browser/provider/index.ts [hunk 1]
```diff
@@ -188,7 +188,7 @@ export default class BrowserProvider {
         await browserTools.maximize(this._getWindowDescriptor(browserId));
     }
 
-    private async _canUseDefaultWindowActions (browserId: string): Promise<boolean> {
+    private async canUseDefaultWindowActions (browserId: string): Promise<boolean> {
```
Comment (AlexKamaev) [inline src/browser/provider/index.ts:191]: should this method be public?

**Comment by AlexKamaev** [comment]
Please add simple description to this PR

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@AndreyBelym FPR

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Pull request merged**